exporter/elasticsearchexporter/model.go

```diff
@@ -449,11 +449,12 @@ func durationAsMicroseconds(start, end time.Time) int64 {
 
 func scopeToAttributes(scope pcommon.InstrumentationScope) pcommon.Map {
 	attrs := pcommon.NewMap()
+
+	scope.Attributes().CopyTo(attrs)
```

Comment:
This reordering has the consequence that if a scope attribute named "name" or "version" exists, it will be overridden by scope name or version. I believe in general this is the right thing to do, with the exception of when the scope name or version is empty.

I'd like to see a follow-up change to only set scope name and version as attributes if they aren't empty.